packages/loader/container-loader/src/audience.ts

```diff
@@ -3,29 +3,40 @@
  * Licensed under the MIT License.
  */
 
-import { EventEmitter } from "@fluid-internal/client-utils";
-import { IAudienceOwner } from "@fluidframework/container-definitions/internal";
+import { TypedEventEmitter } from "@fluid-internal/client-utils";
+import { IAudienceEvents, IAudienceOwner } from "@fluidframework/container-definitions/internal";
 import { assert } from "@fluidframework/core-utils";
 import { IClient } from "@fluidframework/protocol-definitions";
 
 /**
  * Audience represents all clients connected to the op stream.
  */
-export class Audience extends EventEmitter implements IAudienceOwner {
+export class Audience extends TypedEventEmitter<IAudienceEvents> implements IAudienceOwner {
```

Comment:
The class will want its own changeset entry since it lives in a different package.

Reply:
Actually scratch this, the class is not part of the public API surface. Resolving.